hub/core/tensor.py

```diff
@@ -750,6 +752,27 @@ def data(self, aslist: bool = False) -> Any:
                 return list(self.numpy())
             else:
                 return list(map(list, self.numpy(aslist=True)))
+        elif htype == "video":
+            if len(self.index.values) > 2:
```

Comment:
Wherever self.index is accessed, make sure sequence htype is also taken into account. Add test for sequence htype + timestamp as well.